src/corelibs/U2View/src/ov_sequence/PanView.cpp

```diff
@@ -50,50 +50,43 @@
 
 namespace U2 {
 
-/**********************/
-/*PanViewLinesSettings*/
-/**********************/
-
 PanViewLinesSettings::PanViewLinesSettings()
     : numLines(0),
       rowLinesOffset(0),
       showMainRuler(true),
-      showCustomRulers(true),
-      showSequence(true) {
+      showCustomRulers(true) {
 }
 
 int PanViewLinesSettings::getRowLine(int i) const {
-    int firstRowLine = getFirstRowLine();
-    int line = firstRowLine - i + rowLinesOffset;
-    if (line < 0 || line > firstRowLine) {
+    int line = getFirstRowLine() - i + rowLinesOffset;
+    if (line < 0 || line > getFirstRowLine()) {
         return -1;
     }
     return line;
 }
 
 int PanViewLinesSettings::getSelectionLine() const {
-    SAFE_POINT_EXT(showSequence, "Trying to get sequence line, but it's not visible", -1);
     return numLines - 1;
 }
 
 int PanViewLinesSettings::getRulerLine() const {
     SAFE_POINT(showMainRuler, "Trying to get ruler line, but it's not visible", -1);
-    return numLines - 1 - (showSequence ? 1 : 0);
+    return numLines - 2;
 }
 
 int PanViewLinesSettings::getCustomRulerLine(int n) const {
     SAFE_POINT(showCustomRulers, "Trying to get custom ruler line, but it's not visible", -1);
     SAFE_POINT(n >= 0 && n < customRulers.count(), "Invalid number of custom ruler", -1);
 
-    return numLines - 1 - (showSequence ? 1 : 0) - (showMainRuler ? 1 : 0) - n;
+    return numLines - (showMainRuler ? 3 : 2) - n;
 }
 
 int PanViewLinesSettings::getFirstRowLine() const {
-    return numLines - 1 - (showSequence ? 1 : 0) - (showMainRuler ? 1 : 0) - (showCustomRulers ? customRulers.count() : 0);
+    return numLines - 2 - (showMainRuler ? 1 : 0) - (showCustomRulers ? customRulers.count() : 0 );
 }
 
 int PanViewLinesSettings::getAdditionalLines() const {
-    return (showSequence ? 1 : 0) + (showMainRuler ? 1 : 0) + (showCustomRulers ? customRulers.size() : 0);
+    return 1 + (showMainRuler ? 1 : 0) + (showCustomRulers ? customRulers.size() : 0);
```

Comment:
I'm not sure that changed in this file should be reverted. @DmitriiSukhomlinov, can you comment?